book/cli/reth/node.md

```diff
@@ -426,7 +425,7 @@ Logging:
       --log.file.directory <PATH>
           The path to put log files in
           
-          [default: /reth/logs]
+          [default: /home/doni/.config/cache/reth/logs]
```

Comment:
we want it to be a user- and platform-agnostic path

Reply:
5f691ea64f10001264fd9068dd453fbfa9376d4b